pkg/os/launchd/launchd_darwin.go

```diff
@@ -113,15 +114,17 @@ func RestartAgent(label string) error {
 // AgentRunning checks if a launchd service is running
 func AgentRunning(label string) bool {
 	// This command return a PID if the process
-	// is running, otherwise returns "-"
+	// is running, otherwise returns "-" and empty
+	// ouput if the agent is not loaded in launchd
```

Comment:
"ouput" -> "output"
and I think I would say "or empty output" instead of "and .." since we'll either get "-" if loaded but not running, or "" if not loaded.